alfresco-zipkin-repo/src/main/java/eu/xenit/alfresco/instrumentation/solr/representations/SolrRequest.java

```diff
@@ -0,0 +1,57 @@
+package eu.xenit.alfresco.instrumentation.solr.representations;
+
+import brave.Span;
+import org.json.JSONObject;
+
+import java.util.Iterator;
+
+public class SolrRequest {
```

Comment:
Good call to make subclasses to abstract specific kinds of solr requests.

I think we can leverage abstraction a bit better by only calling methods that exist on this base class from the consumers.
That way, we don't need the specific types in `VirtualSolrSpanFactory`.